R/fit.R

```diff
@@ -176,6 +176,7 @@ refit_multiple_optimizers <- function(fit,
 #' @param n_cores (`int`)\cr number of cores to be used.
 #' @param method (`string`)\cr adjustment method for degrees of freedom and
 #'   coefficients covariance matrix.
+#' @param cov (`string`)\cr covariance calculation method.
```

Comment:
```suggestion
#' @param cov (`string`)\cr coefficients covariance matrix adjustment method.
```

Reply:
Given https://github.com/openpharma/mmrm/pull/225 is gave us the new `covariance` argument, I think we should change the name for this one. Because `cov` and `covariance` are too similar and start with the same string. How about `adjust` instead? or `coef_cov`?

Reply:
in summary, we use "covariance" for the structure, and here it is the coefficient coefficient method; so I would prefer `coef_cov` or `coef_cov_method`  instead of `adjust`; for `method`, it is also misleading now (reml = TRUE, and summary shows method = REML previously, now it is updated to inference);

Reply:
yeah `coef_cov` or actually `coef_vcov` or just `vcov` could be good. because with `vcov` it is clear that it is about the coefficients, and it does not conflict with `covariance`, and still short.